apps/server/src/modules/school/controller/dto/migration.response.ts

```diff
@@ -19,6 +19,12 @@ export class MigrationResponse {
 	})
 	oauthMigrationFinished?: Date;
 
+	@ApiPropertyOptional({
+		description: 'Date from when Migration is finally finished and cannot be restarted again',
+		type: Date,
+	})
+	oauthMigrationFinalFinish?: Date;
```

Comment:
The other property is named oauthMigrationFinished. To be consistent this should named oauthMigrationFinalFinished

Reply:
let's keep it that way for now - the rest is done :)